app/src/pages/Lobby/index.tsx

```diff
@@ -137,10 +145,6 @@ function Lobby() {
           <div className={classes.section}>
             <SettingsSection></SettingsSection>
           </div>
-          <Divider variant="middle"></Divider>
-          <div className={classes.section}>
-            <HowToPlay></HowToPlay>
-          </div>
```

Comment:
page getting too long, removed it - not super helpful in my experience playing with different groups of friends